### PR TITLE
convert to simple webpack + npm structure

### PR DESCRIPTION
**Pull request by luksurious**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,36 +0,0 @@
-<!DOCTYPE html>
```
Comment (giardiv) [inline index.html]: Why ???

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+   * Reference: http://webpack.github.io/docs/configuration.html#devserver
+   * Reference: http://webpack.github.io/docs/webpack-dev-server.html
+   */
+  config.devServer = {
+    contentBase: './src/public',
+    stats: 'minimal'
+  };
+
+  return config;
+}();
```
Comment (giardiv) [inline webpack.config.js]: Come ooooooon man !!

File: index.html (hunk 1 above)
Comment (luksurious) [reply index.html]: It's just moved

Comment (giardiv) [reply index.html]: Ahhhhh ok, my error, sorry man

Comment (leirelitw) [reply index.html]: Aha

File: webpack.config.js (hunk 2 above)
Comment (luksurious) [reply webpack.config.js]: I just copied the config, I will need to update it

Comment (giardiv) [reply webpack.config.js]: I hope so...

**Pull request merged**